src/eve/codegen.py

```diff
@@ -58,13 +59,24 @@
 from .visitors import NodeVisitor
 
 
-try:
-    import clang_format
+def _get_clang_format() -> Optional[str]:
+    """Return the clang-format executable, or None if not available."""
+    default_filename = "clang-format"
+    try:
+        import clang_format
+
+        del clang_format
+        return default_filename
+    except ImportError:
+        executable = os.getenv("CLANG_FORMAT_EXECUTABLE", default_filename)
+        ret = run([executable, "--version"])
+        if ret.returncode != 0:
+            return None
+        else:
+            return executable
```

Comment:
```suggestion
    executable = os.getenv("CLANG_FORMAT_EXECUTABLE", "clang-format")
    ret = run([executable, "--version"])
    return executable if ret.returncode == 0 else None
```

Reply:
Since the clang_format Python package is basically empty and it does not provide any other functionality than installing the package, the whole test can be simplified.